examples/webgl_tiled_forward.html

```diff
@@ -0,0 +1,343 @@
+		container.appendChild( renderer.domElement );
+		
+		var renderTarget = new THREE.WebGLRenderTarget();
+
+		scene.add( new THREE.AmbientLight( 0xffffff, 0.33 ) );
+		// At least one regular Pointlight is needed to activate light support 
+		scene.add(new THREE.PointLight( 0xff0000, 0.1, 0.1 ));
+		
+		var bloom = new THREE.UnrealBloomPass( new THREE.Vector2(), 0.8, 0.6, 0.8 );
+		bloom.renderToScreen = true;				
+
+		var stats = new Stats();
+		container.appendChild( stats.dom );
+
+		controls = new THREE.OrbitControls( camera, renderer.domElement );
+		controls.minDistance = 120;
+		controls.maxDistance = 320;
+			
+		var materials = [];
+
+		var Heads = [
+			{ type: "physical", uniforms: { diffuse: 0x888888, metalness: 1.0, roughness: 0.66}, defines: {} },
+			{ type: "standard", uniforms: { diffuse: 0x666666, metalness: 0.1, roughness: 0.33}, defines: {} },
+			{ type: "phong", uniforms: { diffuse: 0x777777, shininess: 20}, defines: {} },
+			{ type: "phong", uniforms: { diffuse: 0x555555, shininess: 10}, defines: { TOON: 1} }
+		];
+
+		function init (geom) {
+			var sphereGeom = new THREE.SphereBufferGeometry( 0.5, 32, 32 );
+			var tIndex = Math.round(Math.random() * 3);
+			Object.keys(Heads).forEach(function(t, index) {
+				var g = new THREE.Group();
+				var conf = Heads[t];
+				var ml = THREE.ShaderLib[conf.type];
+				var mtl = new THREE.ShaderMaterial({
+					lights: true,
+					fragmentShader: ml.fragmentShader,
+					vertexShader: ml.vertexShader,
+					uniforms: THREE.UniformsUtils.clone(ml.uniforms),
+					defines: conf.defines,
+					transparent: tIndex === index ? true : false,
+
+				});
+				mtl.uniforms.opacity.value = tIndex === index ? 0.9 : 1;
+				mtl.uniforms.tileData = State.tileData;
+				mtl.uniforms.tileTexture = State.tileTexture;
+				mtl.uniforms.lightTexture = State.lightTexture;
+				for( var u in conf.uniforms ) {
+					var vu = conf.uniforms[u];
+					if(mtl.uniforms[u].value.set) {
+						mtl.uniforms[u].value.set(vu);
+					} else {
+						mtl.uniforms[u].value = vu;
+					}
+				}
+				mtl.defines["TILED_FORWARD"] = 1;
+				materials.push(mtl);
+				var obj = new THREE.Mesh(geom, mtl);
+				mtl.side = tIndex === index ? THREE.FrontSide : THREE.DoubleSide;
+				
+				g.rotation.y = index * Math.PI / 2;
+				g.position.x = Math.sin(index * Math.PI / 2) * RADIUS;
+				g.position.z = Math.cos(index * Math.PI / 2) * RADIUS;
+				g.add(obj);
+
+				for(var i=0; i < 8; i++) {
+					var sat = Math.floor(33 + 33 * Math.random());
+					var chroma = Math.random() * 100; 
+					var l = new THREE.Group();
+
+					l.add(new THREE.Mesh(
+						sphereGeom, 
+						new THREE.MeshBasicMaterial( { color: new THREE.Color("hsl(" + chroma + ", " + sat + "%, 50%)") } )
+					));
+					l.add(new THREE.Mesh(
+						sphereGeom, 
+						new THREE.MeshBasicMaterial( { 
+							color: new THREE.Color("hsl(" + chroma + ", " + sat + "%, 50%)"), 
+							transparent: true,
+							opacity: 0.033
```

Comment:
is that even visible?

Reply:
Barely, it makes it look like a cheap glow around the light spheres

Reply:
It's nice. On my monitor I had to increase the opacity, though.